src/renderer/shaders/symbolizer/symbolizerVertexShader.glsl

```diff
@@ -65,5 +65,7 @@ void main(void) {
     if (size==0. || (color.a==0. && color != noOverrideColor) || size<orderMinWidth || size>=orderMaxWidth){
         p.x=10000.;
```

Comment:
I remembered when we added the noOverrideColor case, but why do we need to move this position so much? is there no other way to avoid the rendering in those cases?

Reply:
It's a way to ignore the vertex and stop processing it. There is no other way to discard a vertex, this makes sense because normally, it isn't safe to "discard" a vertex since the other 2 vertices may not be discarded. However, for the CVL case, the 3 vertices of a triangle will be discarded or not coherently (no triangles with discarded and not discarded vertices).